packages/lesswrong/components/recommendations/RecommendationsList.tsx

```diff
@@ -20,7 +21,7 @@ const RecommendationsList = ({ recommendations, recommendationsLoading }: Recomm
     {recommendations.map(post =>
       <PostsItem2 post={post} key={post._id}/>)}
     {recommendations.length===0 &&
-      <span>There are no more recommendations left.</span>}
+      <Typography><small>There are no more recommendations left.</small></Typography>}
```

Comment:
I don't love how this looks if the other sections are visible, but I'm guessing this is pretty hard to know at the level of the parent component 

<img width="843" alt="Screen Shot 2020-08-07 at 12 23 51 pm" src="https://user-images.githubusercontent.com/6322017/89641206-19ac0d00-d8a9-11ea-81e9-01eae3cf044a.png">

Reply:
(i.e. there are in fact recommendations left under the recommendations header, they're just provided by other components)

Reply:
I think it's pretty hard to run into this case on a live db. I'll call our work a strict improvement and not try to fix it.